prow/cmd/deck/template/index.html

```diff
@@ -63,14 +63,13 @@
       <table id="builds">
         <thead>
         <tr>
-          <th></th>
-          <th></th>
-          <th></th>
-          <th></th>
+          <th>State</th>
+          <th>Log</th>
+          <th>Rerun</th>
+          <th>Job YAML</th>
           <th>Repository</th>
           <th>Revision</th>
-          <th></th>
```

Comment:
why deleting this column?

Reply:
This relates to removing the splyglass icon as mentioned above.

Reply:
I see, as commented above, I would suggest keeping both

Reply:
fixed!